### ISSUE TITLE
Preferences > Emailing Method > SMTP > Send Test Email 

### ISSUE THREAD

**Issue by BigBoiJefe** [e107inc/e107#2556]
I'd like to request that the email that's entered in the send test email field be saved along with the preferences.

**Comment by arunshekher** [comment]
Yes, its a bit annoying (as it fetches the main admin's email id every time the page refreshes). But doesn't that give us an ever so slight temporal advantage by not adding an additional preference entry to database?
